apps/omg_eth/mix.exs

```diff
@@ -54,15 +54,22 @@ defmodule OMG.Eth.MixProject do
   end
 
   defp contracts_compile do
+    current_path = System.cwd!()
     mixfile_path = __DIR__
-    contracts_dir = Path.join(mixfile_path, "../../deps/plasma_contracts/contracts")
+    contracts_dir = "deps/plasma_contracts/contracts"
 
     contract_paths =
       ["RootChain.sol", "MintableToken.sol"]
-      |> Enum.map(&Path.join(contracts_dir, &1))
       |> Enum.join(" ")
 
-    output_path = Path.join(mixfile_path, "../../_build/contracts")
-    "solc #{contract_paths} --overwrite --abi --bin --optimize --optimize-runs 1 -o #{output_path}"
+    output_path = Path.join([mixfile_path, "../..", "_build/contracts"])
+    compilation_path = Path.join([mixfile_path, "../..", contracts_dir])
```

Comment:
I would also consider renaming this into sth like `sources_dir`

Reply:
I'm just being fussy here

Reply:
ok so,

1/ `compilation_path` is what it says it is - a path from where it is invoking the compilation. Actually it needs to be like that, because otherwise solc goes crazy about the library placeholders (it uses the relative paths from `compilation_path` so it may run out of space... don't ask why)
2/ I'll leave a comment on the above to clarify
3/ `contract_paths` - again this is what it is just the paths are trivial :). I'll think about naming better or commenting

Reply:
I mentioned it as it may be a little confusing